Dockerfile

```diff
@@ -3,11 +3,12 @@ WORKDIR /app
 
 COPY package.json .npmrc* ./
 
-
 COPY lib ./lib
 COPY public ./public
 COPY settings.js ./
 
 RUN npm i
 
+COPY wait_for_postgres.sh /
+ENTRYPOINT [ "/wait_for_postgres.sh" ]
```

Comment:
wait_for_postgres.sh is probably not something that should be included in the production code of this service. I see it more as a tool for easier development. I would suggest moving it back to the Docker compose.

Reply:
Done.